CHANGES.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 2024-01-22 - v0.1.2
+
+- Improved SQL Scheduler UI and add tests
+
+## 2024-01-22 - v0.1.1
```

Comment:
Was there ever a 0.1.1?

Can we also stick to a release format without the `v` prefix, because we don't use it anywhere else...

Reply:
Yes, there was a 0.1.1 with the "first" SQL Scheduler UI and all the stuff included in the list.
About the `v` prefix, my bad with the copy-paste from the `yarn` versioning. I will remove it now!